examples/file-example.php

```diff
@@ -1,10 +1,9 @@
 <?php
 
-use Smartling\Jobs\Params\SearchJobsParameters;
-
 error_reporting(E_ALL);
+
 /**
- * This file contains examples of Smartling API 2.x usage.
+ * This file contains examples of Smartling API 3.x usage.
```

Comment:
Not sure what do you mean API 3.x :). We don't such version in documentation

Reply:
Oh. Will revert these changes in comments.